src/schema/objects/metadata.yaml

```diff
@@ -323,6 +334,33 @@ CellType:
     Values SHOULD come from the
     [cell ontology](http://obofoundry.org/ontology/cl.html).
   type: string
+ChunkTransformationMatrix:
+  name: ChunkTransformationMatrix
+  description: |
+    3x3 or 4x4 affine transformation matrix describing spatial chunk transformation,
+    for 2D and 3D respectively (for examples: `[[2, 0, 0], [0, 3, 0], [0, 0, 1]]`
+    in 2D for 2x and 3x scaling along the first and second axis respectively; or
+    `[[1, 0, 0, 0], [0, 2, 0, 0], [0, 0, 3, 0], [0, 0, 0, 1]]` in 3D for 2x and 3x
+    scaling along the second and third axis respectively).
+    Note that non-spatial dimensions like time and channel are not included in the
+    transformation matrix.
+  type: array
+  minItems: 3
+  maxItems: 4
+  items:
+    type: array
+    minItems: 3
+    maxItems: 4
+    items:
+      type: number
```

Comment:
I think that's a limitation of the type inference function I wrote. It might be best to open an issue about this and tackle it in a separate PR, since I think it has come up before.